src/context.ts

```diff
@@ -48,6 +59,7 @@ export interface BuilderContextType {
   getDocumentTypes: () => Promise<Array<DocumentTypeOption>>;
   getConfidentialityLevels: () => Promise<SelectOption[]>;
   getAuthPlugins: () => Promise<AuthPluginOption[]>;
+  leafletMapBackgrounds: LeafletMapOption[];
```

Comment:
I'd stick to using a callback for consistency with the other callbacks like `getFileTypes`, and also leave 'leaflet' out of the name. If we switch map provider, the tile layer URL will stay the same and then the property names are just confusing.

```suggestion
  getMapTileLayers: () => Promise<MapTileLayerOption[]>;
```

Reply:
Okay, yeah sounds good.

Yeah the leaflet part also doesn't mean a whole lot here.. I'll change the naming in all repositories to reflect this